tests/test_linkify.py

```diff
@@ -69,6 +69,17 @@ def ft(attrs, new=False):
     )
 
 
+def test_invalid_attribute_names():
+    """Test that "invalid-character-in-attribute-name" errors in tokenizing
+    result in attributes with invalid names get dropped.
+
+    """
+    assert (
+        linkify('<a href="http://example.com/"">') ==
```

Comment:
The problem here is the extra " at the end there. That would get turned into an attribute name, so this would end up as:

```
<a "="" href="http://example.com/">
```

Blech.